mits/site_sections/mits_admin.py

```diff
@@ -9,8 +9,96 @@ def index(self, session, message=''):
             'teams': session.mits_teams()
         }
 
+    def create_new_application(self):
+        cherrypy.session.pop('mits_team_id', None)
+        raise HTTPRedirect('../mits_applications/team')
+
     def team(self, session, id, message=''):
         return {
             'message': message,
             'team': session.mits_team(id)
         }
+
+    def set_status(self, session, id, status=None, confirmed=False, csrf_token=None, return_to='index', message=''):
+        team = session.mits_team(id)
+        matching = [t for t in session.mits_teams() if t.name == team.name and t.id != team.id]
+        if confirmed or (status and not matching and team.status == c.PENDING and team.completion_percentage == 100):
+            check_csrf(csrf_token)
+            team.status = int(status)
+            separator = '&' if '?' in return_to else '?'
+            raise HTTPRedirect(return_to + separator + 'message={}{}{}', team.name, ' marked as ', team.status_label)
+
+        return {
+            'team': team,
+            'message': message,
+            'matching': matching,
+            'return_to': return_to
+        }
+
+    def delete_team(self, session, id, duplicate_of=None, csrf_token=None, message=''):
```

Comment:
As mentioned in the description of this PR (and in the docstring on the ``MITSTeam`` model), when we mark a team as soft-deleted, we can optionally mark it as the duplicate of another team so that we can redirect people who log into the deleted team to the correct team's application.